addon/components/zero-clipboard.js

```diff
@@ -4,13 +4,16 @@ import Ember from 'ember';
 export default Ember.Component.extend({
 	attributeBindings: ['title', 'data-clipboard-text', 'data-clipboard-target'],
 	title: 'Copy to clipboard',
+	zeroClipboardEvents: ['ready', 'beforeCopy', 'copy', 'afterCopy', 'destroy', 'error'],
 	didInsertElement: function () {
 	  var client = new ZeroClipboard(this.get('element'));
-
-		//bind aftercopy to an ember event
-	  client.on("aftercopy", Ember.run.bind(this, function(event) {
-			this.send('afterCopy', event);
-	  }));
+		var _this = this;
```

Comment:
Probably want to clean up this whitespace to be consistent with the surrounding code.